src/server/auth/server/testing/oidc_test.go

```diff
@@ -72,15 +75,51 @@ func TestOIDCAuthCodeFlow(t *testing.T) {
 	require.NoError(t, err)
 
 	// Check that pachd recorded the response from the redirect
-	adminClient.Authenticate(adminClient.Ctx(), &auth.AuthenticateRequest{OIDCState: loginInfo.State})
+	_, err = adminClient.Authenticate(adminClient.Ctx(), &auth.AuthenticateRequest{OIDCState: loginInfo.State})
+	require.NoError(t, err)
```

Comment:
This used to be a useless check because we didn't assign the return value from Authenticate to anything. Whoops! We should consider adding a linter that would catch unchecked error returns.

Reply:
My only feedback re:these tests was that 1) ideally a client with no credentials would go through the the auth flow, just to be sure that works, and b) ideally we'd check the token that's returned by using it for `Authorize()` or `WhoAmI()` or something like that.

I pushed a small commit to this branch to make that change